src/msg/async/dpdk/DPDKStack.cc

```diff
@@ -242,6 +242,12 @@ int DPDKWorker::connect(const entity_addr_t &addr, const SocketOptions &opts, Co
   return r;
 }
 
+DPDKStack::~DPDKStack()
+{
+  lderr(cct) << __func__ << " destructing DPDKStack..." << dendl;
```

Comment:
please avoid using `lderr` for non-error messages.

Reply:
ok